articles/article-117-Why-stick-to-one-example.md

```diff
@@ -0,0 +1,29 @@
+<!-- BEGIN TITLE -->
+# Why stick to one example
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+Using the same example, again and again, seems to be quite boring without any added benefit.
+In some instances, this can be used for some positive effect. Using a known example to show different things (like methods or algorithms) saves time, as the people don't have to think about the example used in much detail.
```

Comment:
Language suggestion: Using the same example to show different things ...

Reply:
Did you mean in the third sentence? 
Then we have twice a sentence starting with "Using the same example, .."
If I should I will change it. I was just unsure. That's why I'm asking again.

Reply:
my fault - you are right it is better to leave it the way it is